### PR TITLE
Apply patch from Sebastian Krebs

### PR DESCRIPTION
**Pull request by Thor77**
@KingCrunch

### CODE REVIEW

File: PKGBUILD [hunk 1]
```diff
@@ -1,31 +1,41 @@
 # Maintainer: Thor77 <thor77 at thor77 dot org>
+# Contributor: Sebastian Krebs <sebastian[at]krebs[dot]one>
 
 pkgname=telegraf-bin
 pkgver=1.1.2
 pkgrel=1
```
Comment (kingcrunch) [inline PKGBUILD:6]: You should increase the package version. I guess I missed that in my patch too...
Maybe one should do this during the merge? I'm bot sure.

**Comment by kingcrunch** [comment]
Hi,

I'm back from vacation :)

It looks like everythings there. However, I've left a comment regarding the `pkgrel`.

File: PKGBUILD (hunk 1 above)
Comment (Thor77) [reply PKGBUILD:6]: Nah, you didn't. It was included in your patch but i did not include it on purpose to do it during merge :)

**Comment by Thor77** [comment]
Left that out on purpose, will increase `pkgrel` after merging.

**Pull request merged**